core/src/test/java/com/linecorp/armeria/server/RouteDecoratingServiceTest.java

```diff
@@ -50,6 +63,16 @@ void routeDecorator() throws Exception {
 
         final HttpResponse response2 = webClient.execute(HttpRequest.of(HttpMethod.TRACE, "/"));
         assertThat(response2.aggregate().get().status()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR);
+
+        final WebClient baseContextPathAppliedWebClient = WebClient.of(baseContextPathAppliedServer.httpUri());
+        // This GET request doesn't go through the decorator.
+        final HttpResponse response3 = baseContextPathAppliedWebClient
+                .execute(HttpRequest.of(HttpMethod.GET, "/api"));
+        assertThat(response3.aggregate().get().status()).isEqualTo(HttpStatus.TEMPORARY_REDIRECT);
+
+        final HttpResponse response4 = baseContextPathAppliedWebClient
+                .execute(HttpRequest.of(HttpMethod.TRACE, "/api"));
+        assertThat(response4.aggregate().get().status()).isEqualTo(HttpStatus.TEMPORARY_REDIRECT);
```

Comment:
I expected the status to return `HttpStatus.INTERNAL_SEVER_ERROR`. but it didn't.
And I'm wondering if it will be the correct result based on the specification. 🤔